### PR TITLE
Add/fix help strings to pbench_fio.

### PR DESCRIPTION
**Pull request by ndokos**
Issue #335.

Add help strings for:
  --config
  --max-stddev
  --max-failures
  --install
  --noinstall
  --iodepth

Delete jobs-per-dev from the options: it was not used anywhere.
Fix some typos.

### CODE REVIEW

File: agent/bench-scripts/pbench-fio [hunk 1]
```diff
@@ -108,10 +115,18 @@ function fio_usage() {
 		printf "\t\tnumber of jobs to run, if not given then fio default of numjobs=1 will be used\n"
 		printf -- "\t--samples=<int>\n"
 		printf "\t\tnumber of samples to use per test iteration (default is $nr_sanples)\n"
+	        printf -- "\t--max-stddev=<int>\n"
+                printf "\t\tthe maximum percent stddev allowed to pass\n"
+                printf -- "\t--max-failures=<int>\n"
+                printf "\t\tthe maximum number of failures to get below stddev\n"
+                printf -- "\t--install\n"
+                printf "\t\tinstall only\n"
+                printf -- "\t--noinstall=[n|y]\n"
```
Comment (portante) [inline agent/bench-scripts/pbench-fio]: Why do we have both `--install` and `--noinstall`?

File: agent/bench-scripts/pbench-fio [hunk 2]
```diff
@@ -87,16 +87,23 @@ function fio_usage() {
...
 		printf "\n"
-		printf -- "\t-j str --job-mode=str    str=[serial|concurrent]  (default is $jon_mode)\n"
+		printf -- "\t-j str --job-mode=str    str=[serial|concurrent]  (default is $job_mode)\n"
 		printf "\n"
 		printf -- "\t--ioengine=str           str= any ioengine fio supports (default is $ioengine)\n"
 		printf "\n"
+		printf -- "\t--iodepth=<int>"
+                printf "\t\tSet the iodepth config variable in the fio job file\n"
+		printf "\n"
 		printf -- "\t--clients=str[,str]      str= one or more remote systems to run fio\n"
```
Comment (portante) [inline agent/bench-scripts/pbench-fio]: I believe this also has a `-c` short-hand option.

File: agent/bench-scripts/pbench-fio (hunk 1 above)
Comment (ndokos) [inline agent/bench-scripts/pbench-fio]: I don't know - I proposed getting rid of --noinstall in issue #335.

File: agent/bench-scripts/pbench-fio (hunk 2 above)
Comment (ndokos) [inline agent/bench-scripts/pbench-fio]: But maybe it shouldn't. Aren't we moving away from short options in general?

**Pull request merged**